clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsSubsystemXMLReader.java

```diff
@@ -89,10 +91,17 @@ public void readElement(XMLExtendedStreamReader reader, List<ModelNode> result)
             }
         }
 
+        if (!channelsWithoutExplicitStack.isEmpty()) {
+            ModelNode defaultStack = operation.get(JGroupsSubsystemResourceDefinition.DEFAULT_STACK.getName());
+            for (ModelNode channelOp : channelsWithoutExplicitStack) {
+                channelOp.get(ChannelResourceDefinition.STACK.getName()).set(defaultStack);
+            }
+        }
+
         result.addAll(operations.values());
     }
 
-    private void parseChannels(XMLExtendedStreamReader reader, PathAddress address, Map<PathAddress, ModelNode> operations) throws XMLStreamException {
+    private void parseChannels(XMLExtendedStreamReader reader, PathAddress address, Map<PathAddress, ModelNode> operations, List<ModelNode> channelsWithoutExplicitStack) throws XMLStreamException {
```

Comment:
Is this extra parameter necessary?  We're already collecting operations via the operations parameter.